types/stylelint/index.d.ts

```diff
@@ -235,7 +235,7 @@ declare namespace stylelint {
 	export type RuleContext = {
 		configurationComment?: string | undefined;
 		fix?: boolean | undefined;
-		newline?: string | undefined;
```

Comment:
Is

https://github.com/stylelint/stylelint/blob/131b84f49676578a677d64b3277f66976654d6a9/lib/lintPostcssResult.mjs#L30

always truthy?
If so shouldn't

https://github.com/stylelint/stylelint/blob/131b84f49676578a677d64b3277f66976654d6a9/lib/lintPostcssResult.mjs#L37

shouldn't always be a string?

cf https://github.com/stylelint/stylelint/pull/5418/files#diff-9a4b261e295bef1ccce813aa194ac4023a1dcf98236667889964b1ba6fbc551fR144